ui/src/main/java/edu/wpi/grip/ui/MainWindowView.java

```diff
@@ -190,6 +189,18 @@ public boolean saveProjectAs() throws IOException {
     }
 
     @FXML
+    public void showProjectSettingsEditor() {
+        final ProjectSettings projectSettings = pipeline.getPipeline().getProjectSettings().clone();
```

Comment:
So it copies all of the fields, even if we add more preferences.  I'd rather not have one more thing to remember to update every time we add a new setting